### PR TITLE
Add entity relation diagram for the catalog database

### PR DESCRIPTION
**Pull request by dylanmcreynolds**

### CODE REVIEW

**Comment by padraic-shafer** [comment]
💛

**Comment by dylanmcreynolds** [comment]
I go back and forth on whether to add the fields to the diagram. It makes it big and more likely to get stale. It's great in theory, but developers who are really interested in that level of detail probably have access to tools to look up the fields with. Thoughts?

**Comment by danielballan** [comment]
I second @padraic-shafer's :yellow_heart:.

How about both?

Twice as much to update when things change, but migrations should become fairly rare.

File: docs/source/explanations/architecture.md [hunk 1]
```diff
@@ -124,3 +124,17 @@ providing a writable data store, the Catalog is used.
 [OpenAPI]: https://www.openapis.org/
 [Pydantic]: https://docs.pydantic.dev/
 [content negotiation]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation
+
+
+### Catalog Database
```
[the remark this answers is not in the document]
Comment (danielballan) [reply docs/source/explanations/architecture.md:129]: Actually apparently this is not needed.

**Comment by danielballan** [comment]
Looks good on a local build:

image

**Comment by danielballan** [comment]
If we want to add schemas we can always do that in a follow-up. Merging this now because it is a very useful addition.

**Comment by danielballan** [comment]
If we want to add schemas we can always do that in a follow-up. Merging this now because it is a very useful addition.

**Pull request merged**